### PR TITLE
Tutorial videos with description

### PR DESCRIPTION
**Pull request by erikfTW**
Draft tutorial videos with descriptions.

### CODE REVIEW

File: docs/tutorials/videos/charts.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Taipy Charts to Update Line Types
```
Comment (FlorianJacta) [inline docs/tutorials/videos/charts.md:1]: I don't like this title, I am trying to understand what it means.

File: docs/tutorials/videos/charts.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+Make a line chart for your Taipy application. We're using a basic time series dataset with two variables. You can either use your dataset or get it from the Taipy website.
+
+Charts are visual ways to show and study patterns, trends, and connections in data. Line charts, in particular, show data points connected by straight lines, 
+making it easy to see trends and changes over time. You can change the line styles in charts by adjusting the line properties to create different looks.
+
+To discover more about the Taipy Taipy charts to update line types, click on the tutorial video [here](https://www.youtube.com/watch?v=M32xhZP04yo).
```
Comment (FlorianJacta) [inline docs/tutorials/videos/charts.md:8]: Same comment, I have a hard time with how it is formulated

File: docs/tutorials/videos/markdown-syntax.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+With Taipy GUI, you can easily create a basic application page using Markdown syntax. This covers things like connecting variables and adding visual elements like sliders and charts.
+
+Markdown syntax is a simple way to format plain text and structure documents. It lets you easily add things like headings, lists, links, images, and emphasis 
+to text without needing complex HTML or formatting tools.
+
+Taipy uses [Python Markdown](https://python-markdown.github.io/) to turn your Markdown text into web pages. There are several language extensions that help make your pages look nice and user-friendly. 
+Specifically, Taipy uses the following Markdown extensions: 
+
+- [Admonition](https://python-markdown.github.io/extensions/admonition/)
```
Comment (FlorianJacta) [inline docs/tutorials/videos/markdown-syntax.md:11]: Not sure if this is so interesting but we can keep it

File: docs/tutorials/videos/charts.md (hunk 1 above)
Comment (erikfTW) [reply docs/tutorials/videos/charts.md:1]: Then maybe: "Switching Line Styles with Taipy Charts"

File: docs/tutorials/videos/charts.md (hunk 2 above)
Comment (erikfTW) [reply docs/tutorials/videos/charts.md:8]: Let's try this: 

Charts visualize patterns, trends, and data relationships. Line charts connect data points with straight lines for tracking changes over time. Customize line styles in charts by tweaking line properties for various appearances.

File: docs/tutorials/videos/charts.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/tutorials/videos/charts.md:1]: The title of the video is "Changing line types using Taipy charts".
That to me is fine (of course I would have dropped the Taipy part).
What about you guys?

Of couse... we're not using Taipy charts to change line types, but we have what we have.

File: docs/tutorials/videos/markdown-syntax.md (hunk 3 above)
Comment (FabienLelaquais) [reply docs/tutorials/videos/markdown-syntax.md:11]: Agreed Florian. Every useless detail should be removed. To the point.

File: docs/tutorials/videos/charts.md (hunk 1 above)
Comment (erikfTW) [reply docs/tutorials/videos/charts.md:1]: Well yes, "Changing line types using charts" would work as well, from my point.

File: docs/tutorials/videos/charts.md (hunk 2 above)
Comment (FlorianJacta) [reply docs/tutorials/videos/charts.md:8]: Fine for me!

File: docs/tutorials/videos/charts.md (hunk 1 above)
Comment (FlorianJacta) [reply docs/tutorials/videos/charts.md:1]: I prefer this title, yes!

File: docs/tutorials/videos/markdown-syntax.md (hunk 3 above)
Comment (erikfTW) [reply docs/tutorials/videos/markdown-syntax.md:11]: Then I will remove this section, to keep just information to the point, thanks.

File: docs/tutorials/videos/charts.md (hunk 1 above)
Comment (erikfTW) [inline docs/tutorials/videos/charts.md:1]:
```suggestion
# Changing Line Types Using Charts
```

File: docs/tutorials/videos/charts.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Taipy Charts to Update Line Types
+
+Make a line chart for your Taipy application. We're using a basic time series dataset with two variables. You can either use your dataset or get it from the Taipy website.
+
+Charts are visual ways to show and study patterns, trends, and connections in data. Line charts, in particular, show data points connected by straight lines, 
+making it easy to see trends and changes over time. You can change the line styles in charts by adjusting the line properties to create different looks.
```
Comment (erikfTW) [inline docs/tutorials/videos/charts.md:6]:
```suggestion
Charts visualize patterns, trends, and data relationships. Line charts connect data points with straight lines for tracking changes over time. Customize line styles in charts by tweaking line properties for various appearances.
```

File: docs/tutorials/videos/markdown-syntax.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+Taipy uses [Python Markdown](https://python-markdown.github.io/) to turn your Markdown text into web pages. There are several language extensions that help make your pages look nice and user-friendly. 
+Specifically, Taipy uses the following Markdown extensions: 
+
+- [Admonition](https://python-markdown.github.io/extensions/admonition/) 
+- [Attribute Lists](https://python-markdown.github.io/extensions/attr_list/) 
+- [Fenced Code Blocks](https://python-markdown.github.io/extensions/fenced_code_blocks/) 
+- [Meta-Data](https://python-markdown.github.io/extensions/meta_data/) 
+- [Markdown in HTML](https://python-markdown.github.io/extensions/md_in_html/) 
+- [Sane Lists](https://python-markdown.github.io/extensions/sane_lists/) 
+- [Tables](https://python-markdown.github.io/extensions/tables/)
```
Comment (erikfTW) [inline docs/tutorials/videos/markdown-syntax.md:17]:
```suggestion
```

**Review by FlorianJacta — APPROVED** [review]

**Pull request merged**